src/DataTable/DataTable.scss

```diff
@@ -0,0 +1,109 @@
+
+$table-border: 1px solid $gray-200!default;
+$table-padding-x: 14px;
+$table-padding-y: 14px;
+$table-padding-small: 8px;
+$table-cell-padding: 8px 14px;
+$table-footer-position: flex-end;
+
+.pgn__table-wrapper {
+  border: $table-border;
+}
+
+.pgn__bulk-actions {
+  display: flex;
+}
+
+.pgn__table-container {
+  width: 100%;
+  display: block;
+  overflow-x: auto;
+  -webkit-overflow-scrolling: touch;
+  border-bottom: $table-border;
+}
+
+.pgn__table {
+  width: 100%;
+  border-bottom: $table-border;
+
+  &.is-striped tr:nth-child(even) {
+    background-color: $light-200;
+  }
+  th {
+    background-color: $light-300;
+    padding: $table-cell-padding;
+  }
+  td {
+    white-space: nowrap;
+    text-overflow: ellipsis;
+    overflow: hidden;
```

Comment:
```suggestion
```

Since cells now have a wrapper that does this work, these declarations are no longer needed.